modules/ROOT/pages/dataweave-cookbook-defaults.adoc

```diff
@@ -65,23 +69,45 @@ and
 xref:dataweave-flow-control.adoc#control_flow_else_if[else-if]
 statements to specify a default value.
 
-The following example uses the string `"value is even"` as its default.
+The following example sets `US` as the default value for `userLocation` if it is not present in the JSON input message:
```

Comment:
Below, DITTO previous suggestion of presenting the behavior of a provided value first, and then presenting the default solution for a value not provided.